core/src/test/java/com/linecorp/armeria/common/util/CompositeExceptionTest.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2022 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.common.util;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.any;
+import static org.mockito.Mockito.when;
+
+import java.util.Arrays;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+
+import com.linecorp.armeria.common.Flags;
+
+class CompositeExceptionTest {
+
+    private static final String separator = System.getProperty("line.separator");
+
+    @Mock
+    Sampler<Class<? extends Throwable>> sampler = Flags.verboseExceptionSampler();
+
+    @Test
+    void verboseExceptionEnabledTest() {
+        final IllegalStateException ex1 = new IllegalStateException();
+        final IllegalArgumentException ex2 = new IllegalArgumentException();
+        final CompositeException compositeException = new CompositeException(
+                Arrays.asList(ex1, ex2), sampler);
```

Comment:
We prefer `ImmutableList`.
```suggestion
                ImmutableList.of(ex1, ex2), sampler);
```

Reply:
thanks for review :)